### DISCUSSION TITLE
Installation with Mailcow

### DISCUSSION THREAD

**Discussion by skwiz0d** [the-djmaze/snappymail#782] [Q&A]
Im not sure if anyone has any experience with this but I cant seem to get this to work with my mailcow installation.   I followed along with the install instructions in the wiki and when I try to hit the URL, I get this:

Warning: mkdir(): Permission denied in /web/snappymail/snappymail/v/2.23.1/setup.php on line 73
[202] Data folder permissions error [is_dir]

I do want to mention I tried defaults and the include.php file to set a custom data folder location.  Same error.  I have also verified permissions.

Thank you.

**Comment by MI-KY** [comment]
Hi,
I know that you said you checked the permissions... but https://github.com/the-djmaze/snappymail/blob/a78fcb63082fb46da9353f8a68cd78eb6e790e76/snappymail/v/0.0.0/setup.php#L73 only tries to create a folder.
So the user of your webserver has the correct rights to create folders in the specific path?
